src/prefect/client/client.py

```diff
@@ -1351,82 +1351,6 @@ def set_secret(self, name: str, value: Any) -> None:
         if not result.data.set_secret.success:
             raise ValueError("Setting secret failed.")
 
-    def get_task_tag_limit(self, tag: str) -> Optional[int]:
```

Comment:
Why not update the GraphQL API routes that are being called and leave the client methods with the same name / signature?